lib/screens/home/widgets/grid.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:flutter/material.dart';
```

Comment:
Put these grid related things in their own folder. Should Grid be renamed to something more specific to the home screen? like HomeGrid?

Reply:
I guess we could rename it, though I'm not sure it matters much right now because we don't have any other grids in the app.